docs/mkdocs/docs/api/macros/json_diagnostic_positions.md

```diff
@@ -0,0 +1,51 @@
+# JSON_DIAGNOSTIC_POSITIONS
+
+```cpp
+#define JSON_DIAGNOSTIC_POSITIONS /* value */
+```
+
+This macro enables position diagnostics for generated JSON objects.
+
+When enabled, two new properties: `start_pos()` and `end_pos()` are added to `nlohmann::json` objects and fields. `start_pos()`
+returns the start position of that JSON object/field in the original string the object was parsed from. Likewise `end_pos()` returns the end position of that JSON object/field in the original string the object was parsed from.
```

Comment:
Is the line length for forces line breaks standardized ? Just curious

Reply:
I think I just added it arbitrarily. I will just remove it, since it wraps by default in the viewer anyway